runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/stream/HttpServerFactory.java

```diff
@@ -1610,6 +1710,8 @@ private HttpServer(
             this.delegateNetwork = this::onNetwork;
             this.guard = resolveGuard(binding);
             this.credentials = binding.credentials();
+            this.httpEventRW = new HttpEventFW.Builder();
+            this.eventBuffer = new UnsafeBuffer(ByteBuffer.allocate(1024)); // TODO: Ati
```

Comment:
We might consider creating an `HttpEventContext` that takes `EngineContext` in the constructor, then has methods to log each HTTP event type. This can then be reused across HttpServer and HttpClient.

Reply:
done, lgtm